mobile/ios/Podfile

```diff
@@ -1,6 +1,8 @@
 # Uncomment this line to define a global platform for your project
 platform :ios, '13.0'
 
+source 'https://github.com/ente-io/ffmpeg-kit-custom-repo-ios.git'
```

Comment:
Let's just call it `ffmpeg-kit`?

Reply:
This is repository for our pod repo, I think we can call it pod-repo or similar so we can host other pods as well in future.

Reply:
How was the OG package handling it?

Reply:
They have published it to the official cocoapods repo but I think if we can maintain a custom repo then it would be good cause we are not giving a general package but a stripped out version with our requirements.